exchanges/binanceus/binanceus.go

```diff
@@ -150,11 +150,11 @@ var (
 // GetServerTime this endpoint returns the exchange server time.
 func (bi *Binanceus) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, error) {
 	var response ServerTime
-	return response.Timestamp,
-		bi.SendHTTPRequest(ctx,
-			exchange.RestSpotSupplementary,
-			serverTime, spotDefaultRate,
-			&response)
+	err := bi.SendHTTPRequest(ctx,
```

Comment:
Can one liner this guy

Reply:
If I returned it in one line,  I can not call the Time() method in the return statement because it will return a zero value that was before updating the result by the http call